.docker/build/config.json

```diff
@@ -8,14 +8,16 @@
     "GithubCommitUrl" : "https://github.com/FStarLang/FStar/commit",
     "OnDemandBuildDefinition" : "FStar\\FStar-{agentOS}",
 
-    "DockerFile": ".docker/build/{agentOS}/Dockerfile",
+    "DockerStyle": "run",
+    "DockerFile": ".docker/build/{agentOS}/Dockerfile.build",
     "DependencyFiles" : [
         ".docker/build/build_helper.sh",
         ".docker/build/build.sh"
     ],
+    "DockerCommand": "sudo docker",
```

Comment:
Well, there is by making the user that runs docker part of the docker group. But that is equivalent to making the user root. That's something I'd like to avoid. And running the entire script as root is bad as well.
I don't need `sudo docker` as the default that's checked in here. But the possibility to make it `sudo docker` 🙂

Reply:
I'd prefer the default to be no-sudo and modify `build_local.sh` to be able to override this configuration.